Python/robodk/robolink.py

```diff
@@ -7027,9 +7037,10 @@ def setParam(self, param, value='', skip_result=False):
                 return line
 
 
-if __name__ == "__main__":
-
-    def RoboDKInfo():
+if __name__ == "__main__":    
+    def RoboDKInfo():    
+        return
```

Comment:
Was this causing issues? Are there instances where we run robolink.py on its own other than to debug?

Reply:
Running this module as an executable should not do anything. This was some leftover code by accident from manual tests we did during development.